### PR TITLE
Mobile menu : Finish the hamburger menu 

### PR DESCRIPTION
**Pull request by majdalkilany**
**Create the hamburger menu** 

**Add app.js File** 

**Fix Linters errors**

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @majdalkilany 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that doesn't have a proper PR description. please make sure you summarize the features you've added whenever you create a PR as this gives the reviewer a quick highlight of the new features. 
    image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  menu.classList.toggle('active');
+  body.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-links').forEach((i) => {
+  i.addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    menu.classList.remove('active');
+  });
+});
```
Comment (AbelG101) [inline app.js:16]: - [ ] Ouch! you seem to be missing one of the requirements that states: "When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed." So please make sure you implement that in your JS file here. ✔

File: styles/main.css [hunk 2]
```diff
@@ -551,3 +551,70 @@ footer {
...
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-6px) rotateZ(-45deg);
+  border-bottom: 0.125rem solid #67798e;
+}
+
+.nav-item {
+  list-style: none;
+  margin: 3rem;
+  text-align: center;
+}
```
Comment (AbelG101) [inline styles/main.css:610]: - [ ] Kindly make sure to pay attention to small details such as spacing and alignments of elements as these are the traits of a great front-end developer 
- [ ] As shown in the Figma design below the texts should be aligned to the left & the spacing between the texts is a bit too much so please try to reduce it. ✔
- Figma design:
  image
- Your design:
  Inkedfirefox_zb6b7Q0c9D

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @majdalkilany 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Majd, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job following the instructions of the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**